docs/tutorial/avax-nft.md

```diff
@@ -0,0 +1,399 @@
+---
+title: Mint NFTs on Avalanche 🚧
+description: Learn how to start minting NFTs on Avalanche.
+issueUrl: https://github.com/protocol/nft-website/issues/224
+---
+
+# Mint NFTs on Avalanche
+
+This short tutorial will quickly guide you through getting started with an EVM-compatible NFT minting work flow on [Avalanche](https://www.avax.network/)using Node.js REPL. This is quite useful if you are migrating from Ethereum or other EVM-compatible blockchain and wish to reuse your code.
+
+## Avalanche vs Ethereum
+
+Unlike Ethereum, Avalanche is a network of multi-blockchains, one of which runs a fork of Ethereum Virtual Machine (EVM) and compatible with Ethereum.
+
+Avalanche consists of 3 subnets, X-chain, P-chain, and C-chain.
+
+- **X-chain**: Deals with exchanges of value and runs AVM (namespaced `avm`)
+- **P-chain**: Deals with platform/protocol (core) and able to create new arbitrary blockchains (namespaced `platform`)
+- **C-chain**: the EVM-compatible chain capable of running Solidity smart contract / dapps. It has Ethereum-compatible addresses (hexadecimal strings prefixed with "0x" concatenated with the rightmost 20 bytes of the Keccak-256 hash ECDSA public key)
+
+Most beginner's confusion will be from distinguishing these different subnets. Note that only the C-chain is EVM-compatible and has Ethereum-compatible addresses. and most of dapps will be interacting with this chain.
```

Comment:
Most confusion happens for beginners when trying to distinguish between these different subnets. It's important to note that only the C-chain has EVM-compatibility and Ethereum-compatible addresses, and the majority of dapps will be interacting with this chain.